src/CRUD.php

```diff
@@ -12,12 +12,12 @@ class CRUD extends Grid
     public $formDefault = null;
     public $fieldsDefault = null;
 
-    public $fieldsGrid = null;
+    public $fieldsRead = null;
 
-    public $columnEdit = null;
-    public $fieldsEdit = null;
-    public $formEdit = null;
-    public $pageEdit = null;
+    public $columnUpdate = null;
+    public $fieldsUpdate = null;
+    public $formUpdate = null;
```

Comment:
I think to make code below simpler all defaults can be set here like this:
```
/** @var mixed Form to use when updating record */
public $formUpdate = ['Form', 'layout' => 'FormLayout/Columns'];

/** @var mixed Page to use when updating record */
public $pageUpdate = 'VirtualPage';
// or
public $pageUpdate = ['VirtualPage', 'short_name'=>'edit']; // not sure if short_name should be here or it should be unchangeable and hard-coded somewhere below in code

and so on ...
```

This way it still will be extendable/overwritable, but all such hard-coded "configuration" will be in the beginning of class code so it's easier to understand what's going on here and what property i have to overwrite to get my desired result.

Reply:
Agreed.